addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.xml

```diff
@@ -1,36 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
+    <t t-name="website.s_dynamic_snippet.carousel.arrows">
+        <a t-attf-href="##{uniqueId}" t-attf-class="o_w_carousel_control carousel-control-prev" data-slide="prev" role="button" aria-label="Previous" title="Previous">
+            <span class="fa fa-chevron-circle-left fa-2x"/>
+            <span class="sr-only">Previous</span>
+        </a>
+        <a t-attf-href="##{uniqueId}" t-attf-class="o_w_carousel_control carousel-control-next" data-slide="next" role="button" aria-label="Next" title="Next">
+            <span class="fa fa-chevron-circle-right fa-2x"/>
+            <span class="sr-only">Next</span>
+        </a>
+    </t>
     <t t-name="website.s_dynamic_snippet.carousel">
         <div t-att-id="uniqueId" class="carousel slide" t-att-data-interval="interval">
             <!-- Content -->
+            <t t-set="slideSize" t-value="chunkSize * rowPerSlide" />
```

Comment:
What about renaming `chunkSize` to `rowSize` ? Or if not renamed outside this template, assigning it to a `rowSize` variable ?

Reply:
I'm always careful about renaming variables that were already there. It might break things that I'm not thinking of or customisation done by others. 
However maybe for the sake of readability here, I should do a `t-set="rowSize" t-value="chunkSize"` and replace chunkSize by rowSize in the template. Was that what you're suggesting?

Reply:
Yes, this is what I had in mind.